style.css

```diff
@@ -11,53 +11,120 @@ body {
     color: burlywood;
     display: flex;
     justify-content: center;
-    background: url("pancake.jpg") no-repeat center fixed;
-    background-size: cover;
+    align-items: center;
+    background-color: #fdd39b;
 }
 
 .wrapper {
+    width: 1600px;
     display: flex;
     flex-direction: column;
+    align-items: center;
+    background: url("pancakes2.jpg") no-repeat center fixed;
+    background-size: 1600px;
+    overflow: auto;
+}
+
+.image {
+    display: none;
 }
 
 .main-columns {
```

Comment:
Add `padding-bottom` here to improve how it is displayed on mobile
![image](https://github.com/mikolajkus/Recipe-for-pancakes/assets/15181215/a53f5720-9506-4242-9530-29b89a85cb21)